setup.py

```diff
@@ -26,6 +26,13 @@ def get_version():
     long_description=open('README.rst').read(),
     url='https://github.com/scrapinghub/scrapy-autoextract',
     packages=find_packages(),
+    install_requires=[
+        # FIXME: uncomment after release
+        # 'autoextract-poet',
+        'scrapinghub-autoextract==0.3.0',
```

Comment:
usually pinning versions in install_requires is a bad practice, as it may lead to conflicts easily. E.g. we release  `scrapinghub-autoextract==0.3.1` which is compatible; someone installs scrapy-autoextract, and scrapinghub-autoextract gets downgraded for an user.

Reply:
A rule of thumb: `install_requires` should exclude versions which are known not to work, not specify versions which are known to work.

Reply:
Noted. `'scrapinghub-autoextract==0.3.0'` was replaced by `'scrapinghub-autoextract>=0.3.0'`.